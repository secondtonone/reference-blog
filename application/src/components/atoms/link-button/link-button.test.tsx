import renderer from 'react-test-renderer';
import LinkButton from '.';

describe('Match snapshot LinkButton', () => {
  it('match snapshot default', () => {
    const tree = renderer.create(<LinkButton>link like button</LinkButton>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
