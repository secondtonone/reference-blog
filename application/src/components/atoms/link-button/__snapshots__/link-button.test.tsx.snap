// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Match snapshot LinkButton match snapshot default 1`] = `
<span
  className="link-button__Container-sc-1vjnhcw-0 eyAOse"
  data-test="link-button"
>
  <button
    className="___SButton_rkzvx_gg_ _size_m_rkzvx_gg_ _theme_secondary-muted_rkzvx_gg_"
    data-ui-name="Button"
    onBlur={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onMouseDown={[Function]}
    size="m"
    style={Object {}}
    tabIndex={0}
    type="button"
  >
    <span
      className="___SInner_rkzvx_gg_"
      data-ui-name="Box"
      style={Object {}}
    >
      <span
        className="_size_m_rkzvx_gg_ ___SText_rkzvx_gg_"
        data-ui-name="Button.Text"
        size="m"
        style={Object {}}
      >
        link like button
      </span>
    </span>
  </button>
</span>
`;
